Pidaq/Lib/uploadToDrive/uploadToDrive.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function
+import os
+import logging
+from datetime import datetime
+
+from googleapiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+
+
+SCOPES = 'https://www.googleapis.com/auth/drive.file'
+store = file.Storage('credentials.json') 
+creds = store.get()
+
+if not creds or creds.invalid:
+    flow = client.flow_from_clientsecrets('client_secret.json', SCOPES)
+    creds = tools.run_flow(flow, store)
+    
+DRIVE = build('drive', 'v3', http=creds.authorize(Http()))
+
+
+
+def uploadCsv(filePath, parent):
+    """
+    Uploads a csv file from a local directory to a folder on Google Drive.
+
+    Parameters:
+    "filePath" is the path of the csv file itself and
+                should be passed without quotations.
+    "parent" is the name of the testing folder in the drive
+            and only a specific set of values are allowed.
+    The parent folder is supposedly going to be fixed for a
+    long period of time.
+    """
+    parentDictionary = {'Safety':'1Vnb9UOTatdZUpcBXr1LBjziwNYZzRXQz',
+                        'Integrity': '1Kyr_bJ5MSN3b6w1KMZzdJ7YPyTgPxze1',
+                        'DTS': '100OsZP0JQIk9_VXHXpK8mdhS9GOp3yKm'}
+    parentFolder = parentDictionary[parent]
+    #Dictionary for parent folders and their IDs.
+    #These IDs are to be stored manually.
+
+    
+    date = datetime.today().strftime('%Y-%m-%d')
+    #Today's date in YYYY-MM-DD format is stored in the variable "date"
+   
+    if checkExistingSubFolder(date, parentFolder) == 0:
+        #If the folder does not exist, should be the case
+        #when running for the first time on a given day
+        folder_metadata = {
+            'name' : date,
+            'mimeType': 'application/vnd.google-apps.folder',
+            'parents' : [parentFolder]
+            }
+        folder = DRIVE.files().create(body = folder_metadata,
+                                      fields = 'id').execute()
+        #Creates a folder with the given date
+        folderId = getFolderId(date, parentFolder)
+        #Returns the date folder's ID

```

Comment:
You should put comments above the code that they refer to. Like in a report always talk about a figure before the figure appears:

Instead of what you have, this would be cleaner
```
# Returns the date folder's ID
 folderId = getFolderId(date, parentFolder)
```